translator/cmdutil/translatorutil.go

```diff
@@ -101,16 +101,16 @@ func RunSchemaValidation(inputJsonMap map[string]interface{}) (*gojsonschema.Res
 func checkSchema(inputJsonMap map[string]interface{}) {
 	result, err := RunSchemaValidation(inputJsonMap)
 	if err != nil {
-		panic(err.Error())
+		log.Panicf("E! Failed to run schema validation because of %v", err)
 	}
 	if result.Valid() {
-		fmt.Println("Valid Json input schema.")
+		log.Print("I! Valid Json input schema.")
```

Comment:
Why did this get changed to a log? This introduces more inconsistencies. We use `fmt.Print` for a bunch of things to write to stdout for normal success paths, but this changes this specific function to still log to stderr

Reply:
Mostly changing this would signal us that CWAgent translated successfully and moving on to other executables in the log. I am considering changing most of the fmt to `log.Print` since some could help us in tracking down the issues though.